src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,41 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
```

Comment:
OntologyTerm is will be used by many different modules, its a
core part of metadata.  metadata.advl is going to become types
to support metadata; with the functional area metedata being
part of the appropriate modules.


Reece Hart <notifications@github.com> writes:
> In src/main/resources/avro/metadata.avdl:
> 
> > @@ -9,6 +9,41 @@ protocol Metadata {
> >  import idl "common.avdl";
> >
> >  /**
> 
> My instinct is that OntologyTerm it should go either in alleleAnnotations.avdl
> or perhaps in a separate file.
> 
> —
> Reply to this email directly or view it on GitHub.*
>